src/main/java/me/spartacus04/jext/listeners/JukeboxClickEvent.kt

```diff
@@ -54,8 +58,12 @@ internal class JukeboxClickEvent : JextListener() {
 
     private fun jukeboxGui(event: PlayerInteractEvent, block: Block) {
         event.isCancelled = true
-
-        if(!INTEGRATIONS.hasJukeboxGuiAccess(event.player, block)) return
+        if(INTEGRATIONS.hasJukeboxGuiAccess(event.player, block)==false) {
+            return
+        }
```

Comment:
Just use the ! before the check, this may not make sense now because you made the function nullable but we'll make it not nullable in the next few functions